monix-execution/shared/src/main/scala_2.13+/monix/execution/compat.scala

```diff
@@ -31,5 +31,8 @@ object compat {
     def hasDefiniteSize[X](i: IterableOnce[X]): Boolean = i.knownSize >= 0
 
     def newBuilder[From, A, C](bf: BuildFrom[From, A, C], from: From): mutable.Builder[A, C] = bf.newBuilder(from)
+
+    @inline def toSeq[A](array: Array[AnyRef]): Seq[A] =
+      new scala.collection.immutable.ArraySeq.ofRef(array).asInstanceOf[Seq[A]]
```

Comment:
This is the change

Reply:
It will be hard to keep track of these internal helpers.

Reply:
Any ideas? Alternatively, it could be placed in `reactive` (it has only one usage) but it would be even more scattered then.
I have put it in `private [monix]` in case we can delete it one day.

Initially I wanted to point to the same `ArraySeq` but mutable one doesn't perform well in my benchmarks. It seems like `toSeq` is quite expensive there.

Reply:
That's fine.

Reply:
We'll have to remember we have it I guess.